xbmc/addons/VFSEntry.cpp

```diff
@@ -483,17 +526,19 @@ bool CVFSEntryIDirectoryWrapper::GetDirectory(const CURL& url,
 
 bool CVFSEntryIDirectoryWrapper::DoGetKeyboardInput(void* ctx,
                                                     const char* heading,
-                                                    char** input)
+                                                    char** input,
+                                                    bool hidden_input)
```

Comment:
Thought to do it here, the way how the Keyboard on addon becomes used is complete changed.

How it works has also confused me during test and added direct a documentation part here: https://github.com/AlwinEsch/kodi/blob/0a380b04591da8a99b2fc56c68fa7b3c8f6e3cf6/xbmc/addons/kodi-addon-dev-kit/include/kodi/addon-instance/VFS.h#L278-L330